src/structures/api/SummonerChallenge.ts

```diff
@@ -141,6 +149,8 @@ export class ChallengePreferences {
     this.bannerAccent = data.bannerAccent;
     this.title = data.title;
     this.displayedChallengeIds = data.challengeIds;
+    this.crestBorder = data.crestBorder;
```

Comment:
In the structure, shieldbow should change it to type `'level' | 'ranked'`, and then use the appropriate one here instead of leaving the parsing to the end user.